package.json

```diff
@@ -14,7 +14,8 @@
     "scripts": {
         "pci:runtime": "hygen pci runtime",
         "pci:creator": "hygen pci creator",
-        "pci:install": "hygen pci install"
+        "pci:install": "hygen pci install",
+        "pci:update": "hygen pci update"
```

Comment:
One small comment, I see that you liked to follow the pattern and keep the commands brief.
However, the names do not accurately describe the actions.
If it would have been `pci:create:installer` and `pci:create:migration`, I would not have anything to say.
But it's minor, so keep what you prefer.

Reply:
Good point indeed. I will add these longer names.

BTW, I'd also wanted to turn this tool into a pure CLI, but for the time being I did not find a way to do so without adding a script entry point (npm run scrips does not seem to forward to "binaries").